pallets/phala/src/compute/basepool.rs

```diff
@@ -334,6 +364,12 @@ pub mod pallet {
 			Ok(guard)
 		}
 
+		/// The core procedure that contributes some stake to a pool
```

Comment:
```suggestion
		/// Contributes some stake to the pool
```

The function comment tells the user (i.e. the end user who contribute to the pool) what it does. For more details, please check: https://guiquanz.gitbooks.io/google-cc-style-guide/content/ebook/Comments.html

Reply:
done